taskcluster/ci/ui-test/kind.yml

```diff
@@ -28,7 +28,7 @@ jobs:
             commands:
                 - [wget, {artifact-reference: '<signing/public/build/x86/geckoNightly/target.apk>'}, '-O', app.apk]
                 - [wget, {artifact-reference: '<signing-android-test/public/build/noarch/geckoNightly/target.apk>'}, '-O', android-test.apk]
-                - [automation/taskcluster/androidTest/ui-test.sh, x86, app.apk, android-test.apk, '-1']
+                - [automation/taskcluster/androidTest/ui-test.sh, x86, app.apk, android-test.apk, '50']
```

Comment:
Quick question: does 50 here overrides what's defined in `automation/taskcluster/androidTest/flank-x86.yml`? (not blocking this PR)

Reply:
Yeah we should always use the shard value provided by the templates, which is not happening right now https://github.com/mozilla-mobile/fenix/issues/9185